packages/components/src/components/Input/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
```

Comment:
If we have a choice can we use [reach/visually-hidden](https://reacttraining.com/reach-ui/visually-hidden)?

Reply:
why though? ahah

Reply:
personal preference, no strong reason

Reply:
Changed :)

Reply:
Thanks <3